runtime/doc/lsp.txt

```diff
@@ -2274,6 +2274,26 @@ is_enabled({bufnr})                      *vim.lsp.document_color.is_enabled()*
         (`boolean`)
 
 
+==============================================================================
+Lua module: vim.lsp.linked_editing_range            *lsp-linked_editing_range*
+
+enable({enable})                       *vim.lsp.linked_editing_range.enable()*
+    Enables or disables linked editing ranges
+
+    To "toggle", pass the inverse of `is_enabled()`: >lua
```

Comment:
Maybe others will disagree but I can't think of why someone would want to toggle this. I do think we should keep `enable()` for API consistency, but I would lean towards killing `is_enabled()` (at least for the first iteration of the feature).